### DISCUSSION TITLE
i cant figure out why the built in heart glyph isnt working

### DISCUSSION THREAD

**Discussion by shovelbug** [oraxen/oraxen#998] [Q&A]
it says on startup that is cant find the texture but the texture file is there, shouldnt it also work out of the box?

**Comment by Boy0000** [comment]
use discord for support

**Reply by shovelbug** [reply]
i cant i do not have a phone

**Reply by shovelbug** [reply]
the discord requires a phone to use

**Reply by Boy0000** [reply]
![image](https://github.com/oraxen/oraxen/assets/62521371/24171649-e6c8-47e9-85b1-cb679c647fd4)

**Reply by shovelbug** [reply]
whats your discord?

**Reply by Boy0000** [reply]
![image](https://github.com/oraxen/oraxen/assets/62521371/0b4a6f77-b1db-4eb7-a6c1-08e6ebf868e4)
im at the top just msg from there iguess

**Reply by shovelbug** [reply]
okay i sent friend request

**Reply by Boy0000** [reply]
just send a dm not gonna accept friend requests
if you cant it is because ur not allowing DMs from non-friends

**Reply by shovelbug** [reply]
i cant send dm

**Reply by shovelbug** [reply]
![image](https://github.com/oraxen/oraxen/assets/112828955/a2e1fd9a-4843-4d2e-8db7-8babcf62c7dd)

**Reply by Boy0000** [reply]
because you have DMs blocked from non friends. simply disable that and DM me
